pkg/flowkit/services/transactions.go

```diff
@@ -276,3 +279,39 @@ func (t *Transactions) Send(
 
 	return sentTx, res, err
 }
+
+func (t *Transactions) GetRlp(rlpUrl string) ([]byte, error) {
+
+	client := http.Client{
+		CheckRedirect: func(r *http.Request, via []*http.Request) error {
+			r.URL.Opaque = r.URL.Path
+			return nil
+		},
+	}
+	resp, err := client.Get(rlpUrl)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("error downloading RLP identifier")
+	}
+
+	return ioutil.ReadAll(resp.Body)
+}
+
+func (t *Transactions) PostRlp(rlpUrl string, signed string) error {
```

Comment:
I would accept the `Transaction` as an argument instead of string, this way you encapsulate the logic how to submit a transaction, because otherwise this is more of a function that knows how to send string to server than RLP for transaction.